content/en/seachat/seachat-manual/07-agent-memory.md

```diff
@@ -0,0 +1,338 @@
+> 
+> DO NOT modify the **//QUESTIONS START** and **//QUESTIONS END** tokens. These are crucial to properly populate the memory field for this use case
+
+2. After setting up the description and question section, press the save button at the bottom.
+
+[comment]: ![image](https://github.com/seasalt-ai/ngChat/assets/72329316/9b59ff3a-04aa-44c7-83a8-2cd02fe13f78)
+
+<center>
+<a href="/images/product-updates/seachat/en/memory/question-start.png" target="_blank">
+<img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/memory/question-start.png" alt="">
+</a>
+
+<br/>
+
+*Question Section*
+</center>
+
+3. Head over to the Advanced Settings tab to make sure the fields are properly set.
+    - NOTE: Do NOT change any section in the memory pane when using Survey use cases. If you need to make a change to the questions use the description page.
+
+[comment]: ![image](https://github.com/seasalt-ai/ngChat/assets/72329316/aa6a0fda-3e34-4271-9010-28305d9cd296)
+
+<center>
+<a href="/images/product-updates/seachat/en/memory/advanced-settings.png" target="_blank">
+<img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/memory/advanced-settings.png" alt="">
+</a>
+
+<br/>
+
+*Advanced Settings*
+</center>
+
+# Monitor Conversation Memory
+
+The memory for each conversation will update in real time and we can check the values from the Conversation page
+
+1. Head to the Conversations page from the left hand side
+
+[comment]: ![image](https://github.com/seasalt-ai/ngChat/assets/72329316/00c4d418-879d-49a5-9ffc-a49972668ed7)
+
+<center>
+<a href="/images/product-updates/seachat/en/memory/conversation-page.png" target="_blank">
+<img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/memory/conversation-page.png" alt="">
+</a>
+
+<br/>
+
+*Conversations*
+</center>
+
+2. Find the desired conversation from the list and press the triple dot icon in the bottom right corner for that conversation.
+
+[comment]: ![image](https://github.com/seasalt-ai/ngChat/assets/72329316/9be38868-cdbf-4223-a445-636695b8c922)
+
+<center>
+<a href="/images/product-updates/seachat/en/memory/memory-btn.png" target="_blank">
+<img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/memory/memory-btn.png" alt="">
+</a>
+
+<br/>
+
+*Click on **Memory***
+</center>
+
+3. Pressing "Memory" will bring up the latest extracted values for that conversation
+
+[comment]: ![image](https://github.com/seasalt-ai/ngChat/assets/72329316/5872958e-9b35-4ad7-93c2-50ac2a50f81f)
+
+<center>
+<a href="/images/product-updates/seachat/en/memory/extracted-value.png" target="_blank">
+<img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/product-updates/seachat/en/memory/extracted-value.png" alt="">
+</a>
+
+<br/>
+
+*Latest Extracted Values*
+</center>
+
+## Advanced Memory Use
```

Comment:
remove this section and everything about API. @CJai-K is writing a separate tutorial for the API usages.